### DISCUSSION TITLE
[Question][Hw5] Should we answer question 2 in gradescope submission?

### DISCUSSION THREAD

**Discussion by 33577** [prosyslab-classroom/cs524-program-analysis#308] [Q&A] [answered]
Name: Jonguk Jeon

Hi :) 

I wonder this question is intentionally included in the HW5.
Additionally, I'm unable to find a way to submit the answer to the question; there seems to be only an option for code submission.

Please let me know if there is any misunderstanding. 
Thanks!

<img width="373" alt="image" src="https://github.com/prosyslab-classroom/cs524-program-analysis/assets/34697855/55175d68-22ae-4d5a-8c50-8274ba7245a0">

**Comment by Re-st** [comment, marked as answer]
Please fill out file named 'quiz', which already exists in the directory. (If you can't find it out, refer to the README file where it says 
```
In short, replace all failwith "Not implemented" with your own implementation and give your argument in the quiz file (see below, instruction 5).
```
The 'quiz' will have a hyperlink which probably would lead to the file you need to fill in.

Q2 (질문2) will be graded manually by TAs.

**Comment by 33577** [comment]
Oh I missed it! Thanks for your kind explanation :)
